src/contracts/project.cairo

```diff
@@ -137,6 +139,7 @@ mod Project {
         erc4906: ERC4906Component::Storage,
         #[substorage(v0)]
         metadata: MetadataComponent::Storage,
+        Mint_max_money_amount: u256,
```

Comment:
prefix should be the name of the contract.
Consider putting this variable in the minter?